src/pages/students.js

```diff
@@ -0,0 +1,346 @@
+                    Edition with the token emailed to you to get started.{' '}
+                  </li>
+                </ul>
+              </div>
+            </div>
+            <div className={styles.group}>
+              <p className={styles.question}>Can I purchase Student Edition?</p>
+              <p className={styles.answer}>
+                New Relic offers the Student Edition for FREE to students and
+                educators.
+              </p>
+            </div>
+            <div className={styles.group}>
+              <p className={styles.question}>
+                Can I share my Student Edition with friends?
+              </p>
+              <p className={styles.answer}>
+                The student edition comes with 3 FSO users. Your friends and
+                classmates are also welcome to sign up for their own free
+                account.
+              </p>
+            </div>
+            <div className={styles.group}>
+              <p className={styles.question}>
+                Where can I check the status of my student verification with
+                GitHub?
+              </p>
+              <p className={styles.answer}>
+                In your GitHub account settings, visit Applications > Authorized
+                OAuth Apps to check whether you have authorized the New Relic
+                OAuth app. If you granted this application access to your GitHub
+                account, you should hear back from GitHub as soon as they have
+                verified your student status.
+              </p>
+            </div>
+            <div className={styles.group}>
+              <p className={styles.question}>
+                What if I already have a New Relic free account?
+              </p>
+              <p className={styles.answer}>
+                No problem! Just upgrade your account to the Student Edition. In
+                the main menu, look for Student Edition and follow the
+                instructions.
+              </p>
+            </div>
+            <div className={styles.group}>
+              <p className={styles.question}>
+                What is the difference between the Student Edition and New Relic
+                free account?
+              </p>
+              <div className={styles.answer}>
+                <p>Check out this quick comparison chart</p>
+                <table className={styles.chart}>
+                  <thead>
+                    <tr>
+                      <th>Feature</th>
+                      <th>Student Edition</th>
+                      <th>Free Tier</th>
+                    </tr>
+                  </thead>
+                  <tbody>
+                    <tr>
+                      <td>Free Full Users</td>
+                      <td>3 /month</td>
+                      <td>1 /month</td>
+                    </tr>
+                    <tr>
+                      <td>Free Data</td>
+                      <td>500 gb/m</td>
```

Comment:
Gigabyte should be "GB", capitalization is important here because the lowercase versions actually mean something else. "b" means "bit", not "byte", for example.

Reply:
Also, the more common abbreviation for "per month" is "/mo"